lms/routes.py

```diff
@@ -45,6 +45,21 @@ def includeme(config):
         "blackboard_api.files.via_url", "/api/blackboard/courses/{course_id}/via_url"
     )
 
+    config.add_route(
+        "onedrive.filepicker.authorize",
+        "/api/onedrive/filepicker/authorize",
+    )
+
+    config.add_route(
+        "onedrive.oauth.authorize",
+        "/api/onedrive/oauth/authorize",
+    )
+    config.add_route(
+        "onedrive.oauth.callback",
+        "/api/onedrive/oauth/callback",
+    )
+    config.add_route("onedrive.files.via_url", "/api/onedrive/via_url")
```

Comment:
Ok, so there are two authorization routes: one for the file picker and one for getting file URLs from the API. And there's no list-files route since we're not using our own file picker